app/controllers/orders_controller.rb

```diff
@@ -1,5 +1,28 @@
 class OrdersController < ApplicationController
+  before_action :logged_in_user, only: [:show, :edit, :update, :destroy]
+  include SessionsHelper
+
+  def create
+    order_number = p SecureRandom.hex(2)
```

Comment:
pなくても動くので、消しておいてください！
```suggestion
    order_number = SecureRandom.hex(2)
```

Reply:
上記対応いたしました。
https://github.com/thimon21/railsthimon/pull/11/commits/fd9349d852bba6a944fa4e22f33224530283de05